shuup/admin/modules/settings/__init__.py

```diff
@@ -36,17 +36,60 @@ def __init__(self, model, default_columns, *args, **kwargs):
         self.default_columns = default_columns
         self.ensure_initial_columns(default_columns)
         self.column_spec = self._build_settings_columns()
-        self.columns = [column for column in self.column_spec if self.get_config(column.id)]
+        self._create_active_columns()
+        self._create_inactive_columns()
+
+    def _create_active_columns(self):
```

Comment:
I don't understand the point of these two functions but 99% of the code in them is similar. Refactor please.